src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/NuGet.PackageManagement.VisualStudio.csproj

```diff
@@ -24,7 +24,6 @@
     <PackageReference Include="Microsoft.Build.Utilities.Core" ExcludeAssets="Runtime" PrivateAssets="All" />
     <PackageReference Include="Microsoft.DataAI.NuGetRecommender.Contracts" />
     <PackageReference Include="Microsoft.TeamFoundationServer.ExtendedClient" />
-    <PackageReference Include="Microsoft.VisualStudio.VCProjectEngine" />
```

Comment:
Curious if we know what changed here?

Reply:
The VS SDK references all of those packages.

Reply:
VCProjectEngine is a dependency of Microsoft.VisualStudio.Sdk. We want to use the version that comes with the VS SDK so we're removing the direct reference to VCProjectEngine.

This also reminds me, I also need to remove the entry in Directory.packages.props that specifies the version of VCProjectEngine since that's not used anymore. I'll do that and push an update.